docker/collectd/20-nginx-plus.conf

```diff
@@ -10,7 +10,12 @@
 #
 # System modifications:
 #   None
-
+#
+# Configuration:
+#   * Explicitly set the 'APIVersion' config option to '0' for the 'legacy' versions of NGINX+, if testing needs to be done with the 'legacy' API
+#   * Mention the API base path in the 'APIBasePath' config option, in case the 'status' and 'api'
+#     directives have different API base paths other than defaults('/status' and '/api' respectively)
```

Comment:
```suggestion
#     directives have different API base paths other than defaults ('/status' and '/api' respectively)
```

Reply:
We have added the space and also checked the entire file for similar space-related points and updated the same.